misc/config_tools/schema/checks/rdt_support.xsd

```diff
@@ -81,7 +81,11 @@
     </xs:annotation>
   </xs:assert>
 
-  <xs:assert test="not (//hv//RDT/RDT_ENABLED = 'y' and //hv//SSRAM/SSRAM_ENABLED = 'y')"/>
+  <xs:assert test="not (//hv//RDT/RDT_ENABLED = 'y' and //hv//SSRAM/SSRAM_ENABLED = 'y')">
```

Comment:
PR #7686 added this annotation.  We also need error messages to use the UI names for options not the XML element names.  (Also, there's no way for a developer to edit the RDT_ENABLED element using the configurator.)

I recommend this PR be closed without merging.